src/CheckSum.cs

```diff
@@ -37,6 +70,8 @@ public static uint UpdateCRC32(ReadOnlySpan<byte> bytes, uint initial)
             return crc32;
         }
 
+        internal static readonly int MaxReadBufferSize = 84000;
```

Comment:
Please document the reason for this strange value. 

Also, this should be a `const int`.